php/class-terminus-command.php

```diff
@@ -153,9 +153,14 @@ public static function request($realm, $uuid, $path = FALSE, $method = 'GET', $o
       );
       $cache->put_data($cachekey, $data);
       return $data;
-    } catch( Exception $e ) {
+    } catch( Guzzle\Http\Exception\BadResponseException $e ) {
       $response = $e->getResponse();
       \Terminus::error("%s", $response->getBody(TRUE) );
+    } catch( Guzzle\Http\Exception\BadResponseException $e ) {
+      $request = $e->getRequest();
+      \Terminus::error("Request %s had failed: %s", (string)$request, $e->getMessage() );
```

Comment:
The trouble is that if there is a cURL failure (timeout, certificate, etc.) - different Exception is thrown. Exception which doesn't implement `getResponse`  method. And then crash contains meaningless information that non-existant method cannot be invoked.

I saw failures `Site already exists: <uuid>` which are properly handled by this `getResponse` body - I can extract valuable information then.

So I introduced these two extra tiers to catch different exception types and print corresponding error messages which allows to properly address failure (retry timeouts, notify on other failures, etc.).